docs/02-for-app-authors/10-verification.md

```diff
@@ -2,7 +2,30 @@
 
 ## What is verification?
 
-Verification is a process by which Flathub can confirm that an app is published by its original developer or a third party approved by the developer. This is done by verifying the developers identity using a link to their website or to their profile on a source code hosting site.
+Verification is a process by which Flathub can confirm that an app is published by its original developer or a third party approved by the developer. This is done by verifying the developer's identity using a link to their website or to their account on a source code hosting site. The methods available for verification depend on the app ID.
+
+### Well-known URI
```

Comment:
I guess this heading could be "Website" or something instead, then the well-known URI could just be mentioned in the details below.

Reply:
Yeah, I think that would be better, as we usually call it `Website` and  you already go into the well known below

Reply:
Done!